thunder/executors/nvfuserex_impl.py

```diff
@@ -425,7 +424,7 @@ def compute_tensor_descriptor(
 
 def to_descriptors(proxy_args, args) -> tuple:
     def to_descriptor(proxy_arg, arg):
-        if isinstance(arg, Tensor):
+        if isinstance(arg, torch.Tensor):
```

Comment:
The reason to use `Tensor` directly compared to `torch.Tensor` is an attempt to have lower cost of this function. Let's take a look at what I think is about the average case of 16 inputs to the fusion:
```py
In [1]: import torch

In [2]: from torch import Tensor

In [3]: args = [torch.ones((2, 2)) for _ in range(16)]

In [4]: %%timeit
   ...: [isinstance(arg, Tensor) for arg in args]
   ...:
   ...:
1.15 µs ± 20.1 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
In [5]: %%timeit
   ...: [isinstance(arg, torch.Tensor) for arg in args]
   ...:
   ...:
1.66 µs ± 9.2 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)